packages/suite/src/components/suite/Preloader/Preloader.tsx

```diff
@@ -92,12 +84,8 @@ export const Preloader = ({ children }: PreloaderProps) => {
     if (
         (router.route?.app === undefined ||
             !ROUTES_TO_SKIP_REVISION_CHECK.includes(router.route?.app)) &&
-        selectedDevice?.features &&
-        selectedDevice.connected === true &&
```

Comment:
I think this should fail for remembered wallets, too, so I removed this condition.